homeassistant/components/zwave_js/api.py

```diff
@@ -436,6 +438,50 @@ def filename_is_present_if_logging_to_file(obj: dict) -> dict:
     return obj
 
 
+@websocket_api.require_admin  # type: ignore
```

Comment:
We should use the decorator to check for config entry that we introduced in the other PR.

Reply:
Agreed, that's why I noted that we should merge the other one first 😄